definitions/contested/json/CaseEvent/CaseEvent-prod.json

```diff
@@ -6,7 +6,7 @@
     "Name": "Process Order",
     "Description": "Process Order",
     "DisplayOrder": 36,
-    "PreConditionState(s)": "scheduleRaiseDirectionsOrder",
+    "PreConditionState(s)": "orderDrawn;generalApplication;scheduleRaiseDirectionsOrder",
```

Comment:
I think you don't need to update -prod version. This file will be removed during draft order release.